sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace_hub/workspace_hub.py

```diff
@@ -44,6 +44,7 @@ def __init__(
         public_network_access: Optional[str] = None,
         identity: Optional[IdentityConfiguration] = None,
         primary_user_assigned_identity: Optional[str] = None,
+        enable_data_isolation: bool = False,
```

Comment:
Please double check if optional bools are supposed to be wrapped in Optional[] like the other fields here. Also, please add a docstring somewhere around line 88 below.

Reply:
I just checked - this is correct as it, but it still needs a docstring.